pandas/tests/test_strings.py

```diff
@@ -1208,6 +1209,27 @@ def test_match(self):
         exp = Series([True, np.nan, np.nan])
         tm.assert_series_equal(exp, res)
 
+    def test_fullmatch(self):
+        values = Series(["fooBAD__barBAD", "BAD_BADleroybrown", np.nan, "foo"])
+        result = values.str.fullmatch(".*BAD[_]+.*BAD")
+        exp = Series([True, False, np.nan, False])
+        tm.assert_series_equal(result, exp)
+
+        # Make sure that flags work
+        from re import IGNORECASE
```

Comment:
Don't need this import, just refer to it as `re.IGNORECASE`

Reply:
Can you also split this into a separate test for the flag testing?

Reply:
Thanks for the prompt review! I've removed the unnecessary import, and I've moved the tests of the flags to the existing test case `test_match_findall_flags`. Both changes are in commit 19f605d .